api/api/serializers/image_serializers.py

```diff
@@ -59,13 +60,31 @@ class ImageSearchRequestSerializer(
         required=False,
     )
 
-    def validate_internal__index(self, value):
-        index = super().validate_internal__index(value)
-        if index is None:
-            return None
-        if not index.startswith(IMAGE_TYPE):
-            raise serializers.ValidationError(f"Invalid index name `{value}`.")
-        return index
+
+class ImageSearchRequestSerializer(
+    ImageListRequestSerializer,
+    ImageSearchRequestSourceSerializer,
+    MediaSearchRequestSerializer,
+):
+    """Parse and validate search query string parameters."""
+
+    fields_names = [
```

Comment:
I never noticed it before but the double plural here is a bit strange, `field_names`, like `ImageSearchRequestSourceSerializer` has would be clearer and consistency between all these serializers even better. DRF itself also uses `field_names`, and consistency with the library would be good as well. If we're adding many new instances of the awkward double plural, we might as well just fix the awkwardness, be consistent, and match the library, rather than perpetuate the awkward naming.